source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```
@@ -4675,24 +4686,48 @@ input[type="button"] {
+		pointer-events: none;
+		border: 1px solid var(--brand-orange);
+		border-radius: 5px;
+		/* JS sizes this to the poster's border-box; keep our own border inside
+		   that box so we don't overhang the tile by 2px. */
+		box-sizing: border-box;
+	}
+
 	/* Video Play Overlay */
 	.videoPlayOverlay::after {
 		content: '\f144'; /* Font Awesome play-circle icon */
 		font-family: 'fontAwesome';
```

Stylelint (17.12.0)

[error] Expected no quotes around "fontAwesome" (font-family-name-quotes)

(font-family-name-quotes)